src/root.component.tsx

```diff
@@ -14,29 +31,67 @@ function Root() {
     };
   }, [sidenavOpen]);
 
+  function logout() {
+    authResource
+      .logoutUser()
+      .then(() => setIsLoggedIn(false))
+      .catch(err => {
+        throw err;
+      });
+  }
+
+  if (!isLoggedIn) {
+    // @ts-ignore
+    return <Redirect to={`${window.getOpenmrsSpaBase()}login`} />;
+  }
   return (
-    <nav className={styles.topNav}>
-      <button
-        className="omrs-unstyled omrs-padding-left-4 omrs-padding-right-4"
-        onClick={toggleSidenav}
-      >
-        <svg className={`omrs-icon ${styles.menuIcon}`}>
-          <use xlinkHref="#omrs-icon-menu" />
-        </svg>
-      </button>
-      <div className="omrs-type-title-4">
-        {/* We'll figure out whether to use document.title or not later */}
-        Home
-      </div>
-      <div className={styles.avatar}>AB</div>
-    </nav>
+    <BrowserRouter>
+      <React.Fragment>
```

Comment:
nit: you can do `<>` instead of `<React.Fragment>` for all use cases except when you're adding a `key` prop to the fragment.